runtime/centrifuge/src/lib.rs

```diff
@@ -287,7 +287,7 @@ impl pallet_restricted_tokens::Config for Runtime {
 	type PreFungiblesMutateHold = cfg_traits::Always;
 	type PreFungiblesTransfer = cfg_traits::Always;
 	type PreReservableCurrency = cfg_traits::Always;
-	type WeightInfo = weights::pallet_restricted_tokens::SubstrateWeight<Self>;
+	type WeightInfo = pallet_restricted_tokens::weights::SubstrateWeight<Self>;
```

Comment:
that's right. Until we fix the restricted_tokens benchmarking we disable it for all the runtimes and just use the pallet's weight file directly.

> Does running the individual restricted_tokens pallet weights run successfully?

No because running the benchmarks for a specific pallet still means building and running those benchmarks against one of the runtimes, and it's not listed on any of them with these changes.

So the idea is to just use the current weight as a temporary placeholder until we fix this. Since those weights are relatively high this is fine. As soon as I close this PR I will work on fix those benchmarks properly.